autogalaxy/interferometer/model/analysis.py

```diff
@@ -330,6 +305,34 @@ def fit_interferometer_via_plane_from(
     def fit_func(self):
         return self.fit_interferometer_via_instance_from
 
+    def visualize_before_fit(self, paths: af.DirectoryPaths, model: af.Collection):
+        """
+        PyAutoFit calls this function immediately before the non-linear search begins.
+
+        It visualizes objects which do not change throughout the model fit like the dataset.
+
+        Parameters
+        ----------
+        paths
+            The PyAutoFit paths object which manages all paths, e.g. where the non-linear search outputs are stored,
+            visualization and the pickled objects used by the aggregator output by this function.
+        model
+            The PyAutoFit model object, which includes model components representing the galaxies that are fitted to
+            the imaging data.
+        """
+
+        if paths.is_complete or not os.environ.get("PYAUTOFIT_TEST_MODE") == "1":

```

Comment:
So if it's not test mode we never want to visualise before a fit? That doesn't seem right?